content/docs/rfcs/17/README.md

```diff
@@ -110,6 +110,12 @@ If the `epoch` attached to the message is more than `max_epoch_gap` apart from t
 This is to prevent a newly registered peer from spamming the system by messaging for all the past epochs. 
 `max_epoch_gap` is a system parameter for which we provide some recommendations in section [Recommended System Parameters](#recommended-system-parameters).
 
+**Merkle Root Validation**
+The routing peers MUST check whether the provided merkle root in the RateLimitProof is valid.
+It can do so by maintaining a local set of valid merkle roots, which consist of `acceptable_root_window_size` past roots.
+This allows peers which are not well connected to the network to be able to send messages, accounting for propagation delay. This propagation delay is related to the nature of asynchronous network conditions, which means that peers see membership changes asynchronously, and therefore may have differing local merkle trees.
+`acceptable_root_window_size` is a system parameter for which we provide some recommendations in section [Recommended System Parameters](#recommended-system-parameters)
```

Comment:
I'd be more declarative here, e.g. ~See recommendations on how to choose this

Reply:
Addressed in [3ba4c3c](https://github.com/vacp2p/rfc/pull/539/commits/3ba4c3c75182221f079dab3ec3f935a3464b6803)